src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1172,6 +1159,46 @@ internal override void CreateAsync()
             SendOneItem();
         }
 
+        private void StartOutputAndErrorReaderThreads(Process serverProcess)
+        {
+            ParameterizedThreadStart func = arg => {
+                var tuple = arg as Tuple<StreamReader, bool>;
+                if (tuple == null)
+                {
+                    return;
```

Comment:
Invalid arguments should be checked with Dbg.Assert()?  Should never happen.

Reply:
I can add Dbg.Assert() here, just in case.

Reply:
Add `Dbg.Assert` to cover the invalid argument.